modules/cnv-supported-virtio-drivers.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-installing-virtio-drivers-on-existing-windows-vm.adoc
+// * cnv_users_guide/cnv-installing-virtio-drivers-on-new-windows-vm.adoc
+
+[id="cnv-supported-virtio-drivers_{context}"]
+= Supported VirtIO drivers for Microsoft Windows virtual machines in {ProductName}
+
+.Supported drivers
+|===
+|Driver name | Hardware ID | Description
+
+|*viostor*
+|VEN_1AF4&DEV_1001 +
+VEN_1AF4&DEV_1042
+|The block driver. May represent as an *SCSI Controller* in the *Other devices* 
```

Comment:
Depending on the Windows install, it might display as the listed driver name or it might not. If the latter, then it's likely in the *Other devices* under X controller/device.
Updated to read: "Sometimes displays as a..."